integration-libs/opf/base/root/services/opf-mini-cart-component.service.ts

```diff
@@ -28,21 +21,6 @@ export class OpfMiniCartComponentService extends MiniCartComponentService {
   blockUpdate(decision: boolean) {
     this.isUpdateBlocked = decision;
   }
-  constructor(
-    protected activeCartFacade: ActiveCartFacade,
-    protected authService: AuthService,
-    protected statePersistenceService: StatePersistenceService,
-    protected siteContextParamsService: SiteContextParamsService,
-    protected eventService: EventService
-  ) {
-    super(
-      activeCartFacade,
-      authService,
-      statePersistenceService,
-      siteContextParamsService,
-      eventService
-    );
-  }
```

Comment:
custom constructor not needed - no custom logic and no customization of `super()`'s arguments